test/Sema/package_resilience_access.swift

```diff
@@ -0,0 +1,203 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -emit-module %t/Utils.swift \
+// RUN:   -module-name Utils -swift-version 5 -I %t \
+// RUN:   -package-name mypkg \
+// RUN:   -enable-library-evolution \
+// RUN:   -emit-module -emit-module-path %t/Utils.swiftmodule
+
+// RUN: %target-swift-frontend -typecheck %t/Client.swift -I %t -swift-version 5 -package-name mypkg -verify
+
+// RUN: %target-swift-frontend -emit-sil %t/Client.swift -package-name mypkg -I %t > %t/Client.sil
+// RUN: %FileCheck %s < %t/Client.sil
+
+
+//--- Utils.swift
+
+// Resilient; public. Acessed indirectly.
+public struct PublicStruct {
+  public var data: Int
+}
```

Comment:
Could we apply a similar logic to public decls within the same package? We would still build the public decls as resilient but access them directly, bypassing the resilient indirections for modules of the same package.

Reply:
I'll follow up as discussed offline.